fineract-provider/src/main/java/org/apache/fineract/portfolio/search/data/SearchConditions.java

```diff
@@ -43,18 +44,21 @@ public SearchConditions(final String searchQueryParam, final String searchResour
                 SEARCH_SUPPORTED_RESOURCES.LOANS.name().toLowerCase())) ? true : false;
         this.savingSeach = (null == searchResource || searchResource.toLowerCase().contains(
                 SEARCH_SUPPORTED_RESOURCES.SAVINGS.name().toLowerCase())) ? true : false;
+		this.shareSeach = (null == searchResource || searchResource.toLowerCase().contains(
+                SEARCH_SUPPORTED_RESOURCES.SAVINGS.name().toLowerCase())) ? true : false;
```

Comment:
shareSeach should look for SEARCH_SUPPORTED_RESOURCES.SHARES not SEARCH_SUPPORTED_RESOURCES.SAVINGS